tests/test_server_lifecycle.py

```
@@ -0,0 +1,343 @@
+        def __init__(self, model: Any, processor: Any, **kwargs: Any) -> None:
+            constructs.append((model, processor, kwargs))
+            self.built = True
+
+        @classmethod
+        def from_pretrained(cls, **_kwargs: Any) -> NoReturn:
+            raise AssertionError("from_pretrained() must not be called (double load)")
+
+    fake_pkg = types.ModuleType("starling.qwen3")
+    fake_loader = types.ModuleType("starling.qwen3.loader")
+    fake_loader.load_model_and_processor = (
+        lambda **kwargs: (model_sentinel, processor_sentinel)
```

Ruff (0.16.1)

[warning] Unused lambda argument: `kwargs`

(ARG005)